spec/controllers/openid_connect/configuration_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnect::ConfigurationController do
+  describe '#index' do
+    let(:json_response) { JSON.parse(response.body).with_indifferent_access }
+
+    it 'renders information about the OpenID Connect integration' do
+      get :index
+
+      aggregate_failures do
+        expect(json_response[:issuer]).to eq(root_url)
+        expect(json_response[:authorization_endpoint]).to eq(openid_connect_authorize_url)
+        expect(json_response[:token_endpoint]).to eq(openid_connect_token_url)
+        expect(json_response[:userinfo_endpoint]).to eq(openid_connect_userinfo_url)
+        expect(json_response[:response_types_supported]).to eq(%w(code))
+        expect(json_response[:grant_types_supported]).to eq(%w(authorization_code))
+        expected_loa = [
```

Comment:
unless the OpenId spec somehow differs from SAML in terms of what we support, I'm not sure this is the place to test the specific content. We just want to know it matches our constant, yes? I.e. if the constant changes, this test would needlessly fail.